cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -361,8 +361,10 @@ def install(self):
         # As we don't support installing community as anything other than AIO,
         # not having manager service installed means that this must be premium
         if MANAGER_SERVICE not in config[SERVICES_TO_INSTALL]:
-            yum_install(sources['etcd_rpm_url'])
```

Comment:
Why not add a check for config[MANAGER][PREMIUM_EDITION]?
I think it will be cleaner if more there will be more differences between them

Reply:
That's set during the build process and was only needed for stage. That means that if you're building a custom installer (e.g. to test something) then you get an extra place where bugs can occur.

So, we could use it, but it'd probably be better to just eliminate it because premium-or-not really is not a configuration item.

Reply:
Well, I agree, so maybe add this to some util as a generic solution?

Reply:
After discussion, this has been amended to use a different approach.